openzwavemqtt/util.py

```diff
@@ -0,0 +1,232 @@
+"""Utility functions and classes for OpenZWave."""
+from typing import Any, Dict, List, Union
+
+from .const import (
+    ATTR_LABEL,
+    ATTR_MAX,
+    ATTR_MIN,
+    ATTR_OPTIONS,
+    ATTR_PARAMETER,
+    ATTR_POSITION,
+    ATTR_TYPE,
+    ATTR_VALUE,
+    CommandClass,
+    ValueGenre,
+    ValueType,
+)
+from .exceptions import InvalidValueError, NotFoundError, WrongTypeError
+from .manager import OZWManager
+from .models.node import OZWNode
+from .models.value import OZWValue
+
+
+def get_node_from_manager(
+    manager: OZWManager, instance_id: int, node_id: int
+) -> OZWNode:
+    """Get OZWNode from OZWManager."""
+    instance = manager.get_instance(instance_id)  # type: ignore
+    if not instance:
+        raise NotFoundError(f"OZW Instance {instance_id} not found")
+
+    node = instance.get_node(node_id)
+    if not node:
+        raise NotFoundError(f"OZW Node {node_id} not found")
+
+    return node  # type: ignore
```

Comment:
We should know what we return I think.

Reply:
problem is `get_instance` and `get_node` aren't recognized by mypy because of the way they are created.

Reply:
Can we cast the value to something?